.grabl/automation.yml

```diff
@@ -32,7 +32,11 @@ build:
     test:
       image: vaticle-ubuntu-20.04
       command: |
-        bazel test //...
+        bazel run //:typedb-extractor -- dist/typedb-all-linux
+        sudo systemd-run ./dist/typedb-all-linux/typedb server
+        sleep 60
+        bazel build //...
+        bazel test //... --test_output=streamed --spawn_strategy=local
```

Comment:
We use spawn_strategy=local as pure sandboxed bazel isn't appropriate due to libskiko's aforementioned use of the user's home directory.